src/components/PaintModalHistoryItem.js

```diff
@@ -0,0 +1,19 @@
+const PaintModalHistoryItem = ({data}) => {
```

Comment:
Имя компонента должно быть существительным и предметным
```suggestion
const ModalHistoryItem = ({data}) => {
```